### ISSUE TITLE
About the non-rigid dataset (Fig.5 in your paper)

### ISSUE THREAD

**Issue by yanagi-lmd** [ZENGYIMING-EAMON/CorrNet3D#6]
Thank you for your great contributions! I`m very interested in your paper.

I want to train and test the CorrNet3D model on the non-rigid dataset(Fig.5 in your paper).  May I ask you to explain them and provide the related code, if possible?

**Comment by ZENGYIMING-EAMON** [comment]
Hi, thank you for your notice. This figure is just one of the samples in the dataset provided in the readme. Actually, the pipeline in readme just describe how to train and test the CorrNet3D model on the provided non-rigid dataset (The non-rigid case is the main experiment. And for rigid case, we just random rotate and translate the single shape). Feel free to ask and reopen the issue if you have any further problem running this code.

**Comment by BestSonny** [comment]
@ZENGYIMING-EAMON  I am confused about the description "the pipeline in readme just describe how to train and test the CorrNet3D model on the provided non-rigid dataset". I run the test code of the provided code and data, and it shows that only 100 test samples are provided. I can not link it to the description of your original paper " **For non-rigid shape correspondence, we adopted Surreal [13] as the training dataset, consisting of 230K samples, which were randomly grouped into 115K
training pairs. We conducted the test on the SHREC dataset [9], which has 430 pairs of non-rigid shapes. For rigid shape correspondence, we adopted the training and test dataset splits of the Surreal dataset [13], which contain 230K and 200 samples, respectively, and we randomly rotated and translated the samples to generate 230K pairs and 200 pairs for training and testing, respectively.**"

It seems that the released code is for the rigid dataset, and even so, the number of test samples (100) didn't match the claimed 200 pairs of testing samples (or it is just a typo?).
